src/index.scss

```
@@ -1,5 +1,9 @@
-body {
+html,
+body,
+#root,
```

it looks like redundant work

---

i think you right, i changed this place 
```suggestion
html,
#root,
.wrapper {
  height: 100%;
}
```